library/src/main/kotlin/it/krzeminski/githubactions/actions/madhead/SemverUtils.kt

```diff
@@ -0,0 +1,21 @@
+package it.krzeminski.githubactions.actions.madhead
+
+import it.krzeminski.githubactions.actions.Action
+
+data class SemverUtils(
+    val version: String,
+    val compareTo: String? = null,
+    val satisfies: String? = null,
+    val identifier: String? = null,
+) : Action(name = "madhead/semver-utils@latest") {
```

Comment:
Could we point to concrete major version instead of `latest`?
```suggestion
) : Action(name = "madhead/semver-utils@v1") {
```
I'm figuring that if action's API under your `latest` version changes in a backward-incompatible way, this wrapper will be broken. The idea is to have a separate wrapper class for each major version of each action. It's the theory, let's see how it works in practice :)

BTW, is there any functional difference between the `latest` version and writing just `@main`? According to [the docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsuses), it should work the same.

Reply:
`latest` is a tag, while `main` points to the tip of the `main` branch. They differ.

Reply:
Ok, the [versioned releases](https://github.com/madhead/semver-utils/releases) misled me. It's fine then!